fstab-check.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo "# HELP node_fstab_mount_status List and status of filesystem mountpoints"
```

Comment:
This should be fairly obvious to most people, but...
```suggestion
echo "# HELP node_fstab_mount_status List and status of filesystem mountpoints (0 = not mounted, 1 = mounted)"
```